master/reference/public-cloud/gce.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Calico on Google Cloud
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as
+Google Cloud.
+
+## Why Use Calico in Google Cloud
+
+#### Network Policy for Containers
+
+While Google Cloud Firewall Rules control network traffic sent to VM instances,
+it does not cover containers running on them. Calico automatically implements
+fine-grain, dynamic network policy for containers.
+
+#### No Overlays
```

Comment:
ehhh, this is only true if you're using the GCE backend or a k8s cloud provider integration.

Otherwise yeah we do need an overlay.  Not a super strong point.